src/App.js

```diff
@@ -1,25 +1,27 @@
 import React from 'react';
-import logo from './logo.svg';
+import { Router, Route } from 'react-router-dom'
+import { Grommet } from 'grommet'
+import styled from 'styled-components'
+import history from './history'
+import theme from './theme'
+import Home from './screens/Home'
 import './App.css';
 
+const StyledGrommet = styled(Grommet)`
+  height: 100%
+`
+
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <Router history={history}>
+      <>
+        <main>
+          <StyledGrommet theme={theme}>
```

Comment:
Use `full` instead: https://v2.grommet.io/grommet#full

Reply:
Instead of...?

Reply:
Oh apologies for the half written out thought. Instead of using styled-components to apply 100% height styling. full should do this for you.